libs/utils/android/workloads/vellamo.py

```diff
@@ -0,0 +1,167 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2015, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from android import Workload
+from time import sleep
+
+import os
+import collections
+import copy
+import logging
+
+VELLAMO_DB_PATH = '/data/data/com.quicinc.vellamo/files'
+VELLAMO_SCORE_NAME = 'chapterscores.json'
+VELLAMO_TESTS = ['BROWSER', 'METAL', 'MULTI']
+
+class Vellamo(Workload):
```

Comment:
Post this on a separate patch, maybe in the same patchset of CameraPreview.
Let's have a separate PR for these non WA related WL... they are more easy to merge ;-)